src/edu/csus/ecs/pc2/services/core/JSONUtilities.java

```diff
@@ -12,6 +15,7 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.SerializationFeature;
 
+import edu.csus.ecs.pc2.clics.API202306.CLICSClarification;
```

Comment:
why does this API specific year need to be added?

Reply:
That has since been removed.  The getJsonProperties(Class c) method had previously used "CLICSClarification.class", which was stupid, since the Class was passed in.  I forgot to remove the import in the last push to the PR.  Next push it's fixed.

Reply:
This has been pushed.  Please resolve this comment.